requirements.txt

```diff
@@ -24,7 +24,7 @@ jsonpatch==1.33
 jsonpointer==2.4
 markdown-it-py==3.0.0
 MarkupSafe==2.1.3
-marvin==1.2.1
+marvin==1.5.4
```

Comment:
@vnoelifant, another possible solution: update the marvin version.

Seems like you already did so, but always good to check, if you have the latest version.